src/appleseed/foundation/meta/benchmarks/benchmark_math_filter.cpp

```diff
@@ -33,6 +33,74 @@
 
 using namespace foundation;
 
+BENCHMARK_SUITE(Foundation_Math_Filter_BoxFilter1)
+{
+    struct Fixture
+    {
+        BoxFilter1<float>       m_filter;
+        float                   m_dummy;
+
+        Fixture()
+          : m_filter(2.0f)
+        {
+        }
+    };
+
+    BENCHMARK_CASE_F(Evaluate, Fixture)
+    {
+        m_dummy = 0.0f;
+
+        for (int x = -2; x <= +2; ++x)
```

Comment:
`+2` ? Why not `2`? 💃

Reply:
Out of symmetry with -2 I suppose, I like this notation too.